src/site/content/en/fast/chrome-ux-report-pagespeed-insights/index.md

```diff
@@ -99,23 +111,25 @@ user experiences across all pages of an origin. You can get the same stats for a
 closely aligned with what is available on [BigQuery](/chrome-ux-report-bigquery/), while the page-level
 performance is not made available to query.
 
-{% Img src="image/admin/7Xr7VVdZMbRlEMlgq3cj.png", alt="Origin CrUX performance in PageSpeed Insights", width="800", height="371" %}
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/Mer6ovkOuMxHZH6fhHY7.png", alt="Origin CrUX performance in PageSpeed Insights", width="800", height="426" %}
 
 There is one major difference between the origin-level data on PSI versus
 BigQuery. The datasets on BigQuery are released once a month and encompass the data from the previous calendar month. For example, the 202005 dataset
-includes all user experiences that occurred in May 2020. On the other
-hand, PSI aggregates new data every day encompassing the previous 28 days. So
+includes all user experiences that occurred in the 28 days up until the end of May 2020.
+On the other hand, PSI aggregates new data every day encompassing the previous 28 days. So
 the results you see today may be different tomorrow and they would not
-necessarily be the same as what you'd see in the monthly aggregations on
+necessarily be the same as what you'd see in the current month's aggregations on
 BigQuery.
 
-## Null response when URL not available
+## Responses when URL data not available in CrUX
+
+If the URL you entered is not available in CrUX, PageSpeed Insights will attempt to fallback to original level data as shown below. Clicking on the Information icon will show more explanation.
```

Comment:
Can we embed the ℹ️ icon itself in the text?

Reply:
Just had a second look and it's not an ℹ️  but an exclamation mark:

<img width="253" alt="image" src="https://user-images.githubusercontent.com/10931297/184999017-eed38655-6e53-4dfc-84d2-ef2880b922f6.png">

Reply:
Went with "Clicking on the icon beside the disabled **This URL** button will show more explanation."